rom/rom.go

```diff
@@ -382,6 +385,36 @@ func fileExists(p string, ext ...string) (string, bool) {
 	return p, false
 }
 
+// Not sure if this is the right place for this.
+func convertVideo(p string) error {
+	vidExt := path.Ext(p)
+	baseFile := p[:len(p)-len(vidExt)]
+	outputFile := baseFile + "-converting" + vidExt
+	// Hardcoded command for now, clean this up once we offer more
+	// conversion options.
+	cmd := exec.Command("HandBrakeCLI", "-i", p, "-o", outputFile,
+		"-e", "x264",
+		"-w", "320",
+		"-l", "240",
+		"-r", "30",
+		"--keep-display-aspect",
+		"--decomb",
+		"--audio", "1",
+		"-B", "80",
+		"-E", "av_aac")
+	err := cmd.Run()
+	if err != nil {
+		return err
+	}
+
+	err = os.Rename(outputFile, p)
```

Comment:
Thanks, this has actually been my first Go patch for code that left my own computer. So the feedback is welcome. :)